client/src/pages/ProfileEditPage.js

```diff
@@ -26,6 +26,7 @@ const TitleCenter = styled(Title)`
 const Link = styled(NavLink)`
   position: absolute;
   z-index: 1;
+  text-decoration: none;
 `;
```

Comment:
You are styling the `NavLink` multiple times.  Maybe you can outsource the styled NavLink?